<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>From_FileM</class>
 <widget class="QWidget" name="From_FileM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Файл</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit{
	border: 2px solid #0b79ff;
    border-radius: 6px;
}</string>
         </property>
         <property name="plainText">
          <string>/*
Здесь будет отображаться код вашей программы.
Для начала рисования необходимо после нажатия кнопки &quot;Выбрать файл&quot; указать место расположения нужного файла. После этого нажать кнопку &quot;Записать текст в файл&quot; и затем кнопку &quot;Нарисовать&quot;.
Для редактирования схемы можно изменить текст в форме, после окончания редактирования нажать кнопку &quot;Записать текст в файл&quot;.
*/
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">QScrollArea{
	border: 2px solid #0b79ff;
    border-radius: 6px;
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>575</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pyt_file">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QPushButton:pressed {
	border-color: #8f8f91;
	background-color: rgb(196, 255, 235);
}

QPushButton:hover {
	border-color: rgb(38, 197, 255);
}</string>
         </property>
         <property name="text">
          <string>Выберите файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="zapis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QPushButton:pressed {
	border-color: #8f8f91;
	background-color: rgb(196, 255, 235);
}

QPushButton:hover {
	border-color: rgb(38, 197, 255);
}</string>
         </property>
         <property name="text">
          <string>Записать текст в файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="draw">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QPushButton:pressed {
	border-color: #8f8f91;
	background-color: rgb(196, 255, 235);
}

QPushButton:hover {
	border-color: rgb(38, 197, 255);
}</string>
         </property>
         <property name="text">
          <string>Нарисовать схему</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QPushButton:pressed {
	border-color: #8f8f91;
	background-color: rgb(196, 255, 235);
}

QPushButton:hover {
	border-color: rgb(38, 197, 255);
}</string>
         </property>
         <property name="text">
          <string>Сохранить схему в jpg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="menu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QPushButton:pressed {
	border-color: #8f8f91;
	background-color: rgb(196, 255, 235);
}

QPushButton:hover {
	border-color: rgb(38, 197, 255);
}</string>
         </property>
         <property name="text">
          <string>Выход в главное меню</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Выберите размер кода</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="size_10">
         <property name="text">
          <string>10 строк</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="size_20">
         <property name="text">
          <string>20 строк</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="size_50">
         <property name="text">
          <string>50 строк</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="size_70">
         <property name="text">
          <string>70 строк</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="size_100">
         <property name="text">
          <string>100 строк</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
